smoke-tests/spec/route53_helper.rb

```diff
@@ -0,0 +1,61 @@
+# Create new Route53 zone
+# Expect a domain name in input
+# Return route53 zone object https://docs.aws.amazon.com/sdkforruby/api/Aws/Route53/Types/CreateHostedZoneResponse.html 
+def create_zone(domain)
+    client = Aws::Route53::Client.new()
+    new_zone = client.create_hosted_zone({
```

Comment:
You don't need to assign to a variable and then put the variable at the end of the function, i.e.
```
def foo
  var = do_something
  var
end
```
...is equivalent to
```
def foo
  do_something
end
```

Also, you don't need the outermost `{}` here.
`create_hosted_zone(foo: bar)` is the same as `create_hosted_zone({foo: bar})`